op-acceptance-tests/tests/osaka/osaka_test.go

```diff
@@ -0,0 +1,214 @@
+			l1Config.WithBPO1Offset(0)
+			l1Config.WithL1BlobSchedule(&params.BlobScheduleConfig{
+				Cancun: params.DefaultCancunBlobConfig,
+				Osaka:  params.DefaultOsakaBlobConfig,
+				Prague: params.DefaultPragueBlobConfig,
+				BPO1:   params.DefaultBPO1BlobConfig,
+			})
+		}),
+		sysgo.WithBatcherOption(func(_ stack.L2BatcherID, cfg *batcher.CLIConfig) {
+			cfg.DataAvailabilityType = flags.BlobsType
+		}),
+	)))
+}
+
+func TestBatcherUsesNewSidecarFormatAfterOsaka(gt *testing.T) {
+	t := devtest.SerialT(gt)
+	sys := presets.NewMinimal(t)
+	t.Log("Waiting for Osaka to activate")
+	t.Require().NotNil(sys.L1Network.Escape().ChainConfig().OsakaTime)
+	sys.L1EL.WaitForTime(*sys.L1Network.Escape().ChainConfig().OsakaTime)
+	t.Log("Osaka activated")
+
+	// 1. Wait for the sequencer to build a block after Osaka is activated. This avoids a race
+	//    condition where the unsafe head has been posted as part of a blob, but has not been
+	//    marked as "safe" yet.
+	sys.L2EL.WaitForBlock()
+
+	// 2. Wait for the batcher to include target in a batch and post it to L1. Because the batch is
+	//    posted after Osaka has activated, it means the batcher must have successfully used the
+	//    new format.
+	target := sys.L2EL.BlockRefByLabel(eth.Unsafe)
+	blockTime := time.Duration(sys.L2Chain.Escape().RollupConfig().BlockTime) * time.Second
+	for range time.Tick(blockTime) {
+		if sys.L2EL.BlockRefByLabel(eth.Safe).Number >= target.Number {
+			// If the safe head is ahead of the target height and the target block is part of the
+			// canonical chain, then the target block is safe.
+			_, err := sys.L2EL.Escape().EthClient().BlockRefByHash(t.Ctx(), target.Hash)
+			t.Require().NoError(err)
+			return
+		}
+	}
+}
+
+func TestBlobBaseFeeIsCorrectAfterBPOFork(gt *testing.T) {
+	t := devtest.SerialT(gt)
+	sys := presets.NewMinimal(t)
+	t.Log("Waiting for BPO1 to activate")
+	t.Require().NotNil(sys.L1Network.Escape().ChainConfig().BPO1Time)
+	sys.L1EL.WaitForTime(*sys.L1Network.Escape().ChainConfig().BPO1Time)
+	t.Log("BPO1 activated")
+
+	sys.L1EL.WaitForBlock()
+	l1BlockTime := sys.L1EL.EstimateBlockTime()
+	l1ChainConfig := sys.L1Network.Escape().ChainConfig()
+
+	spamBlobs(t, sys) // Raise the blob base fee to make blob parameter changes visible.
+
+	// Wait for the blob base fee to rise above 1 so the blob parameter changes will be visible.
+	for range time.Tick(l1BlockTime) {
+		info, _, err := sys.L1EL.EthClient().InfoAndTxsByLabel(t.Ctx(), eth.Unsafe)
+		t.Require().NoError(err)
+		if calcBlobBaseFee(l1ChainConfig, info).Cmp(big.NewInt(1)) > 0 {
+			break
+		}
+		t.Logf("Waiting for blob base fee to rise above 1")
+	}
+
+	l2UnsafeRef := sys.L2CL.SyncStatus().UnsafeL2
+
+	// Get the L1 blob base fee.
+	l1OriginInfo, err := sys.L1EL.EthClient().InfoByHash(t.Ctx(), l2UnsafeRef.L1Origin.Hash)
+	t.Require().NoError(err)
+	l1BlobBaseFee := calcBlobBaseFee(l1ChainConfig, l1OriginInfo)
+
+	// Get the L2 blob base fee from the system deposit tx.
```

Comment:
Is it worth making a query to the L1Block predeploy, as another place where we should check for consistency? 

https://github.com/ethereum-optimism/optimism/blob/3ee883bcfb1b6ac4717ae430a84e92d08359c21f/op-e2e/actions/proofs/l1_blob_parameter_forks_test.go#L67-L87